src/interfaces.ts

```diff
@@ -1,5 +1,8 @@
 import type {AxiosResponse} from 'axios';
 import type {DocumentBuilder} from './documentBuilder';
+import type {Transformer} from './validation/transformation/transformer';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
:suspect:

Reply:
Reference in the documentation
![image](https://user-images.githubusercontent.com/12199712/168255566-b3c76ef0-ed30-4aad-a5e4-5517a0a173c0.png)